chia/_tests/util/test_network_protocol_test.py

```diff
@@ -63,7 +61,8 @@ def test_message_ids() -> None:
             message_name = entry.targets[0].id
             if message_id in message_ids:  # pragma: no cover
                 raise AssertionError(
-                    f'protocol message ID clash between "{message_name}" and "{message_ids[message_id]}". Value {message_id}'
+                    f'protocol message ID clash between "{message_name}" and "{message_ids[message_id]}". '
```

Comment:
Side note, use the likes of `{message_name!r}` instead of quoting.  Then you get a good representation instead of maybe something like `"""` if the variable is a `"` for example.  Anyways, unrelated to this pr.  Feel free to close once read.